drivers/scheduler/k8s/k8s.go

```diff
@@ -2929,6 +2936,55 @@ func (k *K8s) getExpectedAutopilotSizeForPvc(pvc *v1.PersistentVolumeClaim, apRu
 	return expectedPVCSize, nil
 }
 
+// ValidateEvents verifies proper alerts and events on resize completion
+func (k *K8s) ValidateEvents(pvc *v1.PersistentVolumeClaim, apRules *apapi.AutopilotRuleList) error {
+
+	// wait until autopilot posts all events
+	time.Sleep(time.Minute)
```

Comment:
The validate implementation should not sleep for this arbitrary period since this is not deterministic. The caller can sleep before invoking ValidateEvents. The implementation of ValidateEvents has no context on how long to sleep.